examples/bri-3/src/bri/zeroKnowledgeProof/services/blockchain/ethereum/ethereum.service.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { EthereumService } from './ethereum.service';
+
+describe('Ethereum services', () => {
+  let ccsm: EthereumService;
+
+  //REMOVE THIS
+  it('empty test', async () => {
+    expect('empty').toBe('empty');
+  });
+
+  //NOTE: Commenting out the test as it requires compiled artifacts to run. RUN THIS.
```

Comment:
Lets just do another issue for tests for our smart contract as part of milestone5